src/resource-graph/azext_resourcegraph/_client_factory.py

```diff
@@ -7,4 +7,8 @@
 def cf_resource_graph(cli_ctx, _):
     from azure.cli.core.commands.client_factory import get_mgmt_service_client
     from .vendored_sdks.resourcegraph import ResourceGraphClient
-    return get_mgmt_service_client(cli_ctx, ResourceGraphClient, subscription_bound=False)
```

Comment:
Why do we remove this?

Reply:
since currently the sdk supports two api versions. The client of preview api version need subscription. It's more like a workaround for existing python sdk. Actually we do have a problem for this workaround. If user don't have right for subscription, he cannot do query anymore. However, they can do it before this PR. But I think it's a sdk issue.